vispy/visuals/isocurve.py

```diff
@@ -96,8 +106,8 @@ def set_data(self, data):
         if _HAS_MPL:
             try:
                 if not (self._X.T.shape == data.shape):
-                    raise
-            except:
+                    raise Exception
+            except Exception:
```

Comment:
Ah, good point, but I suppose I was referring more to the use of a very generic `Exception` in the `raise` which sort of ignores that `self._X.T.shape` may or may not raise something (but maybe that's expected.)